data/hai ships.txt

```diff
@@ -94,6 +164,135 @@ ship "Flea"
 
 
 
+ship "Geocoris"
+	sprite "ship/hai geocoris"
+	attributes
+		category "Heavy Freighter"
+		"cost" 12540000
+		"shields" 13300
+		"hull" 6200
+		"required crew" 15
+		"bunks" 56
+		"mass" 810
+		"drag" 17.1
+		"heat dissipation" 0.45
+		"fuel capacity" 800
+		"cargo space" 761
+		"outfit space" 582
+		"weapon capacity" 172
+		"engine capacity" 94
+		Weapon:
+			"blast radius" 300
+			"shield damage" 3000
+			"hull damage" 1500
+			"hit force" 4500
+	outfits
+		"Pulse Cannon" 2
+		"Pulse Turret" 2
+		"Bullfrog Anti-Missile" 5
+		"Geode Reactor" 2
+		"Hai Valley Batteries"
+		"Hai Diamond Regenerator"
+		"Hai Williwaw Cooling" 5
+		`"Biroo" Atomic Thruster`
+		`"Bondir" Atomic Steering`
+		Hyperdrive
+	engine -64 77
+	engine 64 77
+	gun -25 -109 "Pulse Cannon"
+	gun 25 -109 "Pulse Cannon"
+	turret -47 -73 "Pulse Turret"
+	turret 47 -73 "Pulse Turret"
+	turret -37 6.5 "Bullfrog Anti-Missile"
+	turret 37 6.5 "Bullfrog Anti-Missile"
+	turret -65 25 "Bullfrog Anti-Missile"
+	turret 65 25 "Bullfrog Anti-Missile"
+	turret 0 63 "Bullfrog Anti-Missile"
+	explode "tiny explosion" 50
+	explode "small explosion" 50
+	explode "medium explosion" 50
+	explode "large explosion" 50
+	explode "huge explosion" 20
+	"final explode" "final explosion large"
+	description "The introduction of humans to Hai space caused quite a stir among Hai society, especially in the economy where large human freighters were out competing with the smaller native Hai freighters. In order to prevent humans from taking over large parts of the Hai economy, the Geocoris was developed, a heavy freighter that surpassed anything humanity had to offer at the time."
```

Comment:
Reading over this, `were out competing with the smaller native Hai freighters` should either have the word `out` or `with` removed.

Reply:
I agree and I'll try to fix it as soon as possible